src/test/java/org/jabref/rest/jabmap/resources/RootResourceTest.java

```diff
@@ -0,0 +1,479 @@
+package org.jabref.rest.jabmap.resources;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import javax.ws.rs.client.Client;
```

Comment:
The test does not use [REST-assured](https://github.com/rest-assured/rest-assured) as proposed at https://github.com/SE761Team4/jabref/issues/67#issuecomment-698120868. An ADR should be added for that.